test4044.py

```
@@ -0,0 +1,102 @@
+            find_and_print_matches(response.text, src, domain, regex_patterns, global_matches)
+        except Exception as e:
+            print(f"Error: Unable to fetch the script {src}")
+            print(f"Details: {str(e)}")
+
+    inline_scripts = soup.find_all('script', src=False)
+    for script in inline_scripts:
+        script_content = script.string
+        if script_content:
+            find_and_print_matches(script_content, domain, domain, regex_patterns, global_matches)
+
+def find_and_print_matches(text, src, domain, regex_patterns, global_set):
```

Ruff (0.14.10)

Unused function argument: `src`

(ARG001)